hypervisor/src/hypervisor/scheduler.rs

```diff
@@ -80,11 +79,8 @@ impl<'a> PartitionTimeframeScheduler<'a> {
                 let res = self.partition.run_aperiodic_process(self.timeout);
                 if self.handle_partition_result(res)? == Some(false) {
                     // Aperiodic process was also not run
-                    return Err(LeveledError::new(
-                        SystemError::Panic,
-                        ErrorLevel::Partition,
-                        anyhow!("At least one periodic or aperiodic process is expected to exist"),
-                    ));
+                    let part_name = self.partition.base.name();
+                    warn!("partition {part_name}: no process is scheduled")
```

Comment:
Rather we should check at the end of cold_start if there is no process. If the partition is run multiple times a second, this produces too much log-spam on a relatively high log-level.

Reply:
This then would be a check inside the partition, AFAICT. I'm not an expert on this however.

Reply:
I thought further about this. I think it is completely fine to spam the log if there is a partition that at runtime creates an invalid configurration (no processes...). If anything, this should grab the attention of the developer, so spamming is justified. We could more clever logging (akin "20 identical messages dropped") but I don't see the justification for making this, considering that this only ever occurs when there is a low-key screw up by the developer to begin with.